### ISSUE TITLE
tf2_geometry_msgs missing in noetic

### ISSUE THREAD

**Issue by flabrosse** [ros/ros-overlay#997]
Since melodic seems impossible to get going in gentoo anymore (due to changes in python versions), I'm giving a go at noetic.  I tried to emerge ros-noetic/deskop and get:
```
emerge: there are no ebuilds to satisfy "ros-noetic/tf2_geometry_msgs".
```
The package is indeed missing (as of a sync a few hours ago).

**Comment by flabrosse** [comment]
Thanks @allenh1 for looking into this and adding more packages.  Emerging ros-noetic/desktop now starts.

However, the build fails on the first package (catkin) because it still uses python2_7!  I think it's because it's trying to also install catkin for 2_7 and cannot find catkin_pkg for 2_7 (not supported by the in tree version).  And the version of catkin_pkg will anyway not build for 2_7.  And there are no python targets to be specified on the noetic packages.

**Comment by Uroc327** [comment]
Do I have to sync anything non-standard? As of right now, the repo
```
[ros-overlay]
location = /var/db/repos/ros-overlay
sync-type = git
sync-uri = https://github.com/gentoo-mirror/ros-overlay.git
```
still complains with the same `emerge: there are no ebuilds to satisfy "ros-noetic/tf2_geometry_msgs".` line for me.

**Comment by flabrosse** [comment]
I have this package.  My sync-uri is:
```
sync-uri = https://github.com/ros/ros-overlay
```

**Comment by Uroc327** [comment]
Good catch, thanks!

It's weird, though. The gentoo-mirrors version synced (presumably) just half an hour ago. But it still does not seem to contain `ros-noetic/tf2_geometry_msgs`...

**Comment by Uroc327** [comment]
The gentoo-mirror one has the package now as well. Apparantly the `main` branch caused some issues for the sync.

**Comment by allenh1** [comment]
> The gentoo-mirror one has the package now as well. Apparantly the main branch caused some issues for the sync.

Yeah, that's my mistake. I"m trying to move to `main` as our default branch instead of `master`, but have not figured out a good plan to do so yet. For the time being, please continue syncing with `master`.
